external/enarx-keep-kvm@.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Enarx Keep-Loader (KVM)
+Documentation=https://github.com/enarx/
+
+[Service]
+Type=simple
+Environment="ENARX_BACKEND=kvm"
+ExecStart=/home/mike/programming/enarx-keepldr/target/debug/enarx-keepldr kuuid I%
```

Comment:
Your personal paths don't belong in the repo :-1:

Reply:
Fair - what's the canonical place for this sort of thing?

Reply:
Well, define a placeholder and document, that the path has to be fixed

Reply:
Could be  `@BIN@` `@LOG@` ...